pages/teams/[id].tsx

```diff
@@ -24,6 +25,7 @@ interface Props {
 
 const TeamPage = ({ users, team, forms }: Props): React.ReactElement => {
   return (
+    team ? (
```

Comment:
You can probably get rid of this ternary as the 404 is caught in the `getServerSideProps`.

Reply:
removing the ternary causing this error `Unhandled Runtime Error
TypeError: Cannot read properties of undefined (reading 'map')` in page

Reply:
Interesting, lets try and fix that then...